### PR TITLE
Cpc 437 get bills by owner

### PR DESCRIPTION
**Pull request by Marina1411**
JIRA: link to jira ticket
## Context:
Add access to view all bills by a customer by customer ID
## Changes

- create view bills by customer ID method in api gateway
- add a bill by customer ID page to the UI
- add a button to view bills by customer ID to owner details and to bill history

## Before and After UI (Required for UI-impacting PRs)
image
image
image

## Dev notes (Optional)
- Specific technical changes that should be noted
## Linked pull requests (Optional)
- pull request link

### CODE REVIEW

**Review by ElricVermaut — APPROVED** [review]

**Comment by swafit** [comment]
In the pr you mention you have changed some UI elements yet no pictures were provided for a before and after of the UI. Please remember to add images to the pr if UI changes are made to help visualize the difference.

**Review by swafit — DISMISSED** [review]
Overall this looks great. I made a few comments on naming conventions that you should check out and on angular linking methodology.

File: api-gateway/src/main/resources/static/scripts/app.js [hunk 1]
```diff
@@ -8,8 +8,8 @@ const whiteList = new Set([
 /* App Module */
 var petClinicApp = angular.module('petClinicApp', [
     'ui.router', 'layoutNav', 'layoutFooter', 'layoutWelcome', 'ownerList', 'ownerDetails', 'ownerForm', 'petForm'
-    , 'visits', 'vetList','vetForm','vetDetails', 'loginForm', 'rolesDetails', 'signupForm', 'billDetails', 'billHistory'
-    , 'verification' , 'adminPanel', 'visitList','billForm']);
```
Comment (swafit) [inline api-gateway/src/main/resources/static/scripts/app.js:12]: Could you clarify why visitList and billForm were removed?

**Review by swafit — COMMENTED** [review]
I need to understand why visitList and billForm were reviewed before I can approve this request.

File: api-gateway/src/main/resources/static/scripts/app.js (hunk 1 above)
Comment (Marina1411) [reply api-gateway/src/main/resources/static/scripts/app.js:12]: fixed

File: api-gateway/src/main/resources/static/scripts/app.js [hunk 2]
```diff
@@ -9,7 +9,8 @@ const whiteList = new Set([
 var petClinicApp = angular.module('petClinicApp', [
     'ui.router', 'layoutNav', 'layoutFooter', 'layoutWelcome', 'ownerList', 'ownerDetails', 'ownerForm', 'petForm'
     , 'visits', 'vetList','vetForm','vetDetails', 'loginForm', 'rolesDetails', 'signupForm', 'billDetails', 'billsByOwnerId', 'billHistory'
-    , 'verification' , 'adminPanel']);
+    , 'verification' , 'adminPanel' , 'verification' , 'adminPanel', ]);
```
Comment (swafit) [inline api-gateway/src/main/resources/static/scripts/app.js:12]: code is duplicated and visitList & billForm are missing (removed accidentally in previous commit)

**Review by swafit — COMMENTED** [review]
Please review the code and make the changes commented.

**Pull request closed without merging**